src/RTT_Format_Reader/CellDefs.cc

```diff
@@ -84,75 +79,67 @@ void CellDefs::readEndKeyword(ifstream & meshfile)
  *        definitions.
  */
 void CellDefs::redefineCellDefs(
-    vector_vector_int const & cell_side_types,
-    std::vector< std::vector< std::vector< size_t > > >
-    const & cell_ordered_sides)
-{
-    Insist(cell_side_types.size()
-           == static_cast<size_t>(dims.get_ncell_defs()),
-	   "Error in supplied cell redefinition side types data.");
-    Insist(cell_ordered_sides.size()
-           == static_cast<size_t>(dims.get_ncell_defs()),
-	   "Error in supplied cell redefinition ordered side data.");
+    vector_vector_int const &cell_side_types,
+    std::vector<std::vector<std::vector<size_t>>> const &cell_ordered_sides) {
+  Insist(cell_side_types.size() == static_cast<size_t>(dims.get_ncell_defs()),
+         "Error in supplied cell redefinition side types data.");
+  Insist(cell_ordered_sides.size() ==
+             static_cast<size_t>(dims.get_ncell_defs()),
```

Comment:
The indentation rule is applied to the first argument to `Insist `in exactly the same way as it would be applied to the argument w/o `Insist`.  This is way I was trying to say above. If this still isn't clear, we can talk about it tomorrow.